### DISCUSSION TITLE
WAF加载封锁记录异常

### DISCUSSION THREAD

**Discussion by fxln0** [1Panel-dev/1Panel#4919] [Q&A]
### 1Panel 版本

1.10.6-lts

### 问题描述

WAF加载封锁记录显示今天封锁的记录显示已解封，重启服务器后恢复正常封禁中
![image](https://github.com/1Panel-dev/1Panel/assets/168253884/7590f7a3-f94d-4b69-a1c4-ce407f71c581)
![image](https://github.com/1Panel-dev/1Panel/assets/168253884/527f748b-e73a-45c1-b3dc-e096104d23d6)
![image](https://github.com/1Panel-dev/1Panel/assets/168253884/5e088e1f-a94f-4a86-8e29-d17027e9631b)

### 重现步骤

登录1panel立即查看waf封锁记录和拦截记录

**Comment by zhengkunwang223** [comment]
我们先复现一下

**Comment by fxln0** [comment]
好的

**Comment by zhengkunwang223** [comment]
这个封锁为临时封锁 缓存在 OpenResty 中的 重启 OpenResty 会丢失 所以显示解封
